### DISCUSSION TITLE
WOCHE 06 - Aktuelle Hürden

### DISCUSSION THREAD

**Discussion by DossohMael** [mausio/corgi-shop-doc#8] [Blogs]
Servus!
Die Entwicklung des Front-sowie des Backends schreiten voran. 
Jedoch stießen wir letzte Woche auf folgende Schwierigkeiten, die die Entwicklung bis jetzt verlangsamt haben.

*Im Backend:*

1. **Datenbankintegration:**
Die Datenbankintegration lief bis jetzt sehr gut, jedoch gestaltet sich das Design der Implementierung von Merchandise-Kategorien und Items schwieriger als erwartet. Die Konfiguration der Initialisierung und Verbindung zur Datenbank führten zu Problemen, die noch nicht ganz gelöst werden konnten.

2. **DTO-Konvertierung:**
Die Konvertierung zwischen Datenbankentitäten und DTOs bereitet noch Schwierigkeiten. 

3. **Testing:**
Tests konnten also noch nicht durchgeführt werden.

*Im Frontend:*

Hier hat die Erweiterung der Limitierung der maximalen Bitlänge für Einträge in die Datenbank zu unerwarteten Problemen geführt. Dadurch konnte die bestehende Datenbank nicht mehr korrekt interpretiert werden. Bis die Ursache der Fehlermeldungen uns aufgefallen ist, sind mehrere Tage vergangen.

Wir hoffen euch ab jetzt wieder Neuerungen vorstellen zu können.
Bis bald!

**Comment by lukas-mchr** [comment]
Schade, dass es bei euch aufgrund der Probleme im Backend aktuell zu einer langsameren Entwicklung kommt. Ist aber schön zu hören, dass ihr auf einem guten Weg seid, die Probleme in den Griff zu bekommen. 

Kleiner Typo btw: Habt am Ende einmal Datenban geschrieben ;-)

**Comment by Phylyp1** [comment]
Testen wird überbewertet ;)
Spaß, sehr guter Blogbeitrag, auch wenn gerade nicht alles so läuft wie geplant
